CodePush.m

```diff
@@ -56,6 +57,11 @@ + (NSURL *)bundleURLForResource:(NSString *)resourceName
     NSError *error;
     NSString *packageFile = [CodePushPackage getCurrentPackageBundlePath:&error];
     NSURL *binaryJsBundleUrl = [[NSBundle mainBundle] URLForResource:resourceName withExtension:resourceExtension];
+    NSDictionary *binaryFileAttributes = [[NSFileManager defaultManager] attributesOfItemAtPath:[binaryJsBundleUrl path] error:nil];
```

Comment:
Could we refactor this logic out into a utils method called `[CodePushUtils binaryBundleModifiedDateString]` or something? That way, this code and the download code can both use them. I'm assuming you're storing this date as a way to share this value between both methods?